Cannot find source file: .../src/pyne/pyne.cpp

rev. 472ac87 fails like this when being configured with ```-DBUILD_STATIC_LIBS=OFF -DBUILD_TESTS=OFF```

Git submodule is present but it doesn't contain ```pyne.cpp```.

OS: FreeBSD 13

> Any updates on this @yurivict?

The error is still present in the current master branch:
```
-- Configuring done
CMake Error at cmake/DAGMC_macros.cmake:228 (add_library):
  Cannot find source file:

    /disk-samsung/freebsd-ports/science/dagmc/work/DAGMC-3fd335e/src/pyne/pyne.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc
Call Stack (most recent call first):
  src/pyne/CMakeLists.txt:36 (dagmc_install_library)


CMake Error at cmake/DAGMC_macros.cmake:228 (add_library):
  No SOURCES given to target: pyne_dagmc-shared
Call Stack (most recent call first):
  src/pyne/CMakeLists.txt:36 (dagmc_install_library)
```

The code does refer to ```pyne.cpp``` (please grep for it), but it doesn't contain ```pyne.cpp```.

--- a/doc/CHANGELOG.rst
+++ b/doc/CHANGELOG.rst
@@ -11,7 +11,7 @@ Next version
 
    * adding BUILD_EXE option (default ON) allowing to build only the dagmc libs without the executable (for static and/or shared libs) (#717)
    * Including installation of a CMake version file for use with `find_package` in client codes. (#722)
-   * CMake option to checkout PyNE submodule automatically (#734)
+   * CMake option to checkout PyNE submodule automatically (#734, #787)
    * GitHub Action to build and upload Docker images. (#746, #748, #754, #757, #758, #759, #765, #767)
    * Enforcing usage of Python3 for PyNE amalgamation. (#773)
    * Adding workflow_dispatch option to docker_publish workflow (#776)

--- a/src/pyne/amalgamate_pyne.sh
+++ b/src/pyne/amalgamate_pyne.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # take 2 arguments:
 # $1 location of the pyne source folder

